src/js/components/HeaderBar.tsx

```diff
@@ -5,19 +5,18 @@ import ClusterDropdown from "./ClusterDropdown";
 import AccountDropdown from "./AccountDropdown";
 
 export default function HeaderBar() {
-  // remove this to activate component
-  if (arguments) {
```

Comment:
Logo Link & Sidebar are still missing - as far as I understood we can not enable this before these two things are done :(